docs/README_sp.md

```diff
@@ -2,6 +2,8 @@
 
 [English](../README.md) | **Español**
 
+## 🛑 Este proyecto ha sido archivado y no mantiene actualmente 🛑
```

Comment:
```suggestion
## 🛑 Este proyecto se ha archivado y no se mantiene actualmente 🛑
```

Reply:
Could also do "está archivado" instead of "se ha archivado"

Reply:
Done